modules/compiler/spirv-ll/include/spirv-ll/module.h

```diff
@@ -362,6 +363,26 @@ class Module : public ModuleHeader {
   ///
   const std::string &getSourceMetadataString() const;
 
+  /// @brief Sets the string used to hold the process/processor
+  void setModuleProcess(const std::string &str);
+
+  /// @brief Gets the string used to hold the process/processor
+  const std::string &getModuleProcess() const;
+
+  /// @brief Check if this ID is an OpExtInst with the given opcode.
+  ///
+  /// @return Returns true if this opcode is an OpExtInst with the given
+  /// opcode, false otherwise.
+  bool isOpExtInst(spv::Id id, uint32_t opcode,
```

Comment:
I find the name confusing with the fact that is seems to check it is an OpeExtInst and then checks against the opcode as something extra.

Reply:
Yeah, I couldn't think of a better name. It's a bit confusing because there are two opcodes going on. `OpExtInst` is itself an opcode enumeration, but each `OpExtInst` has an operand which denotes the specific opcode of the extended instruction.